### PR TITLE
Add TypeScript documentation

### PR DESCRIPTION
**Pull request by screendriver**
Related to issue #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,49 +4,87 @@
...
+
+If you're using [TypeScript](http://typescriptlang.org) in your build you have to modify your
+webpack configuration as follows:
+
+```js
+module: {
+  rules: [
+    {
+      test: /\.worker\.ts$/,
+      use: ['workerize-loader', 'ts-loader'],
```
Comment (developit) [inline README.md:55]: is this correct? `ts-loader` should already be getting applied via line 59.

Comment (leonidkuznetsov18) [reply README.md:55]: this solutions not work for me

Comment (johnnyreilly) [reply README.md:55]: > is this correct? ts-loader should already be getting applied via line 59.

I'm guessing that the purpose here is to ensure that `.worker.ts` are chained through ts-loader and then workerize-loader?

Comment (screendriver) [reply README.md:55]: To be honest I can't remember. It's over one year ago when I opened that pull request 😅 The only thing I can remember is that I had some issues and I copy and pasted my solution into this pull request and it worked.

Comment (johnnyreilly) [reply README.md:55]: Does the solution in your pr work for you? I'm thinking of taking it for a spin (I happened upon this in the first place as I'm planning to use web workers with TypeScript and webpack)

Comment (screendriver) [reply README.md:55]: The last time when I played around with that was 1 year ago. So I don't know if it still works 🤷‍♂️

**Pull request closed without merging**